KnowSheet/scripts/check-headers.sh

```diff
@@ -10,9 +10,16 @@ set -u -e
 CPPFLAGS="-std=c++11 -g -Wall -W -DBRICKS_CHECK_HEADERS_MODE"
 LDFLAGS="-pthread"
 
+if [ $(uname) = "Darwin" ] ; then
+  CPPFLAGS+=" -stdlib=libc++ -x objective-c++ -fobjc-arc"
```

Comment:
This is bad idea, as mobile clang compilers for iOS and Android still does not support it.
I would prefer to avoid using thread_local for at least a year.

> On Mar 3, 2015, at 13:55, Maxim Zhurovich <notifications@github.com> wrote:
> 
> In KnowSheet/scripts/check-headers.sh <https://github.com/KnowSheet/Bricks/pull/108#discussion_r25678021>:
> 
> > @@ -10,9 +10,16 @@ set -u -e
> >  CPPFLAGS="-std=c++11 -g -Wall -W -DBRICKS_CHECK_HEADERS_MODE"
> >  LDFLAGS="-pthread"
> >  
> > +if [ $(uname) = "Darwin" ] ; then
> > +  CPPFLAGS+=" -stdlib=libc++ -x objective-c++ -fobjc-arc"
> To maximize support of c++11 on Mac. In particular, thread_local specifier can be used if clang is installed from macports (unfortunately default XCode's clang still doesn't support it)
> http://stackoverflow.com/a/23850891 <http://stackoverflow.com/a/23850891>
> —
> Reply to this email directly or view it on GitHub <https://github.com/KnowSheet/Bricks/pull/108/files#r25678021>.
>